src/styles/base/colors.scss

```diff
@@ -1,3 +1,3 @@
-$title-color: #C21703;
+$title-and-game-color: #C21703;
```

Comment:
color naming is one of the hardes thing in project... 
What if there will be a third element (fourth, fifth...) added? :D 

I think, that if color is reused in couple of places we can:
- use the color name (in exaple some-red)
- or if . the color is main in design, maybe `main-color` ?

Reply:
Dear my beloved Codereviewer, for now I understand well all the colors, but in the future of my project, if something turns out less understandable, I would rename it immediately. 

Yours, forever, js noobie